youtube_dl/extractor/vk.py

```diff
@@ -315,6 +333,12 @@ def _real_extract(self, url):
                 m_rutube.group(1).replace('\\', ''))
             return self.url_result(rutube_url)
 
+        dailymotion_urls = DailymotionIE._extract_urls(info_page)
+        if dailymotion_urls:
+            return self.playlist_result([
```

Comment:
It does not look like it's possible by design to embed several videos on a video page at vk. So just use `url_result` on the first URL.